src/clarity/forms/demo/form-fields.demo.scss

```diff
@@ -1 +1,18 @@
+/*!
```

Comment:
@youdz: since we are adding the copyright in a multi line comment, will it be included in the compiled file multiple times? just want to double check. should we use single line comments instead?